examples/OSPolicyAssignments/console/exec_script_linux_all.yaml

```diff
@@ -1,17 +1,16 @@
-# This policy uses an `exec` resource to conditionally create a file.
-# When the policy is applied to a target vm, the `validate` step first runs.
-# If the target file already exists, the `validate` step returns exit code `100` to indicate that the resource is already in the desired state. In this case, the `enforce` step will not be run.
-# If the target file does not exist, the `validate` step returns exit code `101` to indicate that the resource is not in the desired state. In this case, the `enforce` step will be run.
-# The `enforce` step brings the resource to its desired state by creating the file and returns exit code `100` to indicate that the enforcement was successful.
-id: policy-to-exec-script
+# An OS policy that ensures Apache web server is running on Linux OSes.
+id: apache-always-up-policy
 mode: ENFORCEMENT
 resourceGroups:
   - resources:
-      id: touch-file
+      id: ensure-apache-is-up
       exec:
         validate:
           interpreter: SHELL
-          script: if ls /tmp/test_file >/dev/null; then exit 100; else exit 101; fi
+          # If Apache web server is already running, return an exit code 100 to indicate that exec resource is already in desired state. In this scenario, the `enforce` step will not be run.
```

Comment:
Consider respecting an 80 or 100 char line length so folks with small screens can read the comments w/out horizontal scrolling. (here and elsewhere)

Reply:
Done